app/models/asset.rb

```diff
@@ -29,6 +28,8 @@ class Asset < ApplicationRecord
   scope :hottest,         -> { where('hotness > 0').order('hotness DESC') }
   scope :most_commented,  -> { where('comments_count > 0').order('comments_count DESC') }
   scope :most_listened,   -> { where('listens_count > 0').order('listens_count DESC') }
+  scope :with_user,       -> { joins(:user).where('users.id IS NOT NULL') }
```

Comment:
I believe this is a workaround because we are having trouble "reaching" users from an asset when they are deleted. I think the answer is to unscope the default user scope on `belongs_to :user` below, adding `.with_deleted` — I'm not 100% sure how that affects the counter cache.....I guess we want the `assets_count` to decrement when a track is deleted — maybe we have specs in place for this...